phpBB/phpbb/db/migration/data/v30x/release_3_0_5_rc1.php

```diff
@@ -130,4 +136,12 @@ public function remove_duplicate_auth_options()
 			}
 		}
 	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function setContainer(ContainerInterface $container = null)
```

Comment:
I think this should be part of the migration, instead of having to repeat it over and over again.
Maybe just have two abstract classes `\phpbb\db\migration\migration` and `\phpbb\db\migration\migration_container_aware`